src/NuGet/Microsoft.VisualStudio.LanguageServices.nuspec

```diff
@@ -34,5 +34,8 @@
     <file src="Dlls\ServicesVisualStudio\Microsoft.VisualStudio.LanguageServices.xml" target="lib\net46" />
     <file src="Dlls\ServicesVisualStudio\Microsoft.VisualStudio.LanguageServices.pdb*" target="lib\net46" />
     <file src="$thirdPartyNoticesPath$" target="" />
+
+    <!-- Satellite assemblies -->
+    <file src="Dlls\ServicesVisualStudio\**\Microsoft.VisualStudio.LanguageServices.resources.dll" target="lib\net46" />
```

Comment:
Not strictly necessary, since in this case this package is really a reference-assembly only package -- we could technically put the binary asset into ref/ and it'd be fine. But this is fine if you prefer the consistency.

Reply:
In that case I'll just revert this file. There's no real point in including the satellite assemblies if we don't mean for this package to be used for redistribution.